src/instance.js

```diff
@@ -38,7 +57,11 @@ export default class Instance {
 		// so that it's rerendered every time, not just new static parts, like in non-debug mode
 		this.fullStaticOutput = '';
 
-		this.container = reconciler.createContainer(this.rootNode, false, false);
+		if (options.experimental) {
```

Comment:
Could you add this to `index.d.ts`, too? Thanks! :)

Reply:
Good catch!